rele/config.py

```diff
@@ -22,6 +23,7 @@ def __init__(self, setting):
         self.app_name = setting.get("APP_NAME")
         self.sub_prefix = setting.get("SUB_PREFIX")
         self.middleware = setting.get("MIDDLEWARE", default_middleware)
+        self.encoder = setting.get("ENCODER", DEFAULT_ENCODER)
```

Comment:
This should work like the middleware setting: a dotted path lazy import. Otherwise you'll have to import the class into your settings file.